### DISCUSSION TITLE
El-Upload 报错

### DISCUSSION THREAD

**Discussion by habaoyf** [element-plus/element-plus#8532] [Q&A]
El-Upload 报错 elment-plus 2.0.2版本正常
通过el-upload 自动上传文件，带Data属性

![image](https://user-images.githubusercontent.com/45748106/176407399-2a78e146-90e5-44df-a827-284b25faa9ee.png)

**Comment by sxzz** [comment]
https://antfu.me/posts/why-reproductions-are-required-zh
https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md
